blib2to3/pytree.py

```diff
@@ -592,17 +666,17 @@ def __init__(self, type=None, content=None, name=None):
         if type is not None:
             assert type >= 256, type
         if content is not None:
-            assert not isinstance(content, str), repr(content)
-            content = list(content)
-            for i, item in enumerate(content):
+            # assert not isinstance(content, str), repr(content)
```

Comment:
What's wrong with this assertion?

Reply:
It caused mypy to barf because it then couldn't figure out what type `content` (and thus `newcontent`) was. The actual issue is that the type of `content` was wrong, though. I'll fix that, though all this code is dead in black.